lib/mongo/session.rb

```diff
@@ -676,6 +676,7 @@ def pin(server)
         raise ArgumentError, 'Cannot pin to a nil server'
       end
       if Lint.enabled?
+        # TODO: does any work need to be done here?
         unless server.mongos?
```

Comment:
This relates to my question in the PR description -- what should I do about server selection?